fluentd/vendored_gem_src/fluent-plugin-prometheus/lib/fluent/plugin/in_prometheus_tail_monitor.rb

```diff
@@ -9,7 +9,7 @@ class PrometheusTailMonitorInput < Fluent::Plugin::Input
 
     helpers :timer
 
-    config_param :interval, :time, default: 5
+    config_param :interval, :time, default: 1
```

Comment:
Can you explain why we need to move to more frequent interval ?

Reply:
This if needs to be changed should be made in the config generated for the plug-in, not here

Reply:
all changes done in the vendored_gem_src are restored back as it is not supposed to be changed for creating custom plugin. now all changes sit in fluentd/lib/ folder.